plugin/web/extensions/sidebar/core/components/content/common/DatasetCard/Field/Fields/general/HeightReference.tsx

```diff
@@ -0,0 +1,72 @@
+import { Field } from "@web/extensions/sidebar/core/components/content/common/DatasetCard/Field/common";
+import { Wrapper } from "@web/extensions/sidebar/core/components/content/common/DatasetCard/Field/commonComponents";
+import { Select } from "@web/sharedComponents";
+import { useCallback, useState } from "react";
+
+import { BaseFieldProps } from "../types";
+
+const heightReferenceOptions = [
+  { value: "clamp", label: "Clamp to ground" },
+  { value: "relative", label: "Relative to ground" },
+  { value: "none", label: "None" },
```

Comment:
I think we have the same labels (or close to it in Re:Earth. Can you update the labels to be Japanese?